pages/my-events/index.js

```diff
@@ -14,18 +26,20 @@ export default function MyEvents({ ownEvents, onSave }) {
 
   return (
     <>
-      <h1>My Events List</h1>
-      <MyEventsList
-        ownEvents={storedOwnEvents}
-        onDeleteEvent={handleDeleteEvent}
-        onSave={onSave}
-      />
-      <Link href="/"> Back to homepage </Link>
-      <br></br>
-      <br></br>
-      <Link href="/add-event">
-        <button>Add your Event</button>
-      </Link>
+      <Wrapper>
+        <PageTitle>My Events </PageTitle>
+        <MyEventsList
+          ownEvents={storedOwnEvents}
+          onDeleteEvent={handleDeleteEvent}
+          onSave={onSave}
+        />
+        <Link href="/"> Back to homepage </Link>
+        <br></br>
+        <br></br>
+        <Link href="/add-event">
+          <button>Add your Event</button>
+        </Link>
```

Comment:
Nesting a `<button> `inside a `<Link> `is semantically incorrect and can lead to unpredictable behavior because both elements are interactive and intended for user actions. <Link> is used for navigation, while `<button>` is for actions. Please adjust. 

```suggestion
        <Link href="/add-event">Add your Event </Link>
```